.travis.yml

```diff
@@ -2,8 +2,6 @@ language: php
 
 php:
   - 5.3
-  - 5.4
-  - 5.5
```

Comment:
The idea was restart the travis build for catch latest HHVM versions and monitor the progress,